all-imports.html

```diff
@@ -0,0 +1,12 @@
+<!--
```

Comment:
iron-component-page has all kinds of problems that I'm hoping to address soon, and this is one of them. In the CLI templates we've just removed it for now, redirecting to the demo. Docs will be served up via polyserve and/or webcomponents.org